minio/console
# Use swagger api for Add KMS Key

Updates AddKeyForm.tsx to add KMS Key.

### Test Steps:
1. Enable encryption like:
```
export MINIO_KMS_KES_ENDPOINT=https://play.min.io:7373                                                     
export MINIO_KMS_KES_KEY_FILE=root.key
export MINIO_KMS_KES_CERT_FILE=root.cert
export MINIO_KMS_KES_KEY_NAME=my-minio-sse-kms-key
```
2. Run minio
3. In UI go to Administration/Encryption
4. Click on Create Key
5. Type a new key and click on save
6. New key should be added and listed
